### DISCUSSION TITLE
Can SEB send log report to moodle?

### DISCUSSION THREAD

**Discussion by ghost** [SafeExamBrowser/seb-win-refactoring#325] [Q&A]
Hello Devs,
I know SEB records a log on students' computers. So how do SEB send log files to moodle? I want to see if my students use a second monitor, third-party software, red lock screen…. Can I
Thank you.

**Comment by dbuechel** [comment]
SEB does not transmit any data to other servers, so you can't do this. With [SEB-Server](https://github.com/SafeExamBrowser/seb-server) however, you'd be able to retrieve that information (and much more).

**Reply by dbuechel** [reply]
Please always create a new discussion for new or unrelated questions. Yes, as the message resp. the options on the lock screen indicate, once you unlock SEB you're thereby allowing the current configuration. In an exam situation, students shouldn't know the unlock / quit password.
